tools/benchmark_tool/openvino/tools/benchmark/utils/utils.py

```diff
@@ -757,23 +757,65 @@ def get_network_batch_size(inputs_info):
 def show_available_devices():
     print("\nAvailable target devices:  ", ("  ".join(Core().available_devices)))
 
+def dict2string(config):
+    ret = "{"
+    for k, v in config.items():
+        if isinstance(v, dict):
+            sub_str = "{"
+            for sk, sv in v.items():
+                if isinstance(sv, bool):
+                    sv = str(sv).capitalize()
+                sub_str += "{0}:{1},".format(sk, sv)
+            sub_str = sub_str[:-1]
+            sub_str += "}"
+            ret += "{0}:{1},".format(k, sub_str)
+        else:
+            ret += "{0}:{1},".format(k, v)
+    ret = ret[:-1]
+    ret += "}"
+    return ret
+
+def string2dict(device_properties_str):
+    config = {}
+    if not device_properties_str:
+        return config
+    if device_properties_str[0] != '{' or device_properties_str[-1] != '}':
+        raise Exception(
+            "Can't deterimine batch size: batch is different for different inputs!")
```

Comment:
It seems exception is not related to the problem

Reply:
Updated.